hmda/Jenkinsfile

```diff
@@ -6,34 +6,31 @@ podTemplate(label: 'buildSbtPod', containers: [
 volumes: [
   hostPathVolume(mountPath: '/var/run/docker.sock', hostPath: '/var/run/docker.sock'),
 ]) {
-   node('buildSbtPod') {
-     def repo = checkout scm
-     def gitCommit = repo.GIT_COMMIT
-     def gitBranch = repo.GIT_BRANCH
-     def gitDescribe = sh(returnStdout: true, script: 'git describe --always --tags')
-     def gitTag = sh(returnStdout: true, script: "git tag --sort version:refname | tail -1").trim()
-     def commitId = sh(returnStdout: true, script: 'git rev-parse HEAD')
-     def shortCommit = commitId[0..7]
-     def result = sh(returnStatus: true, script: "git log -1 | grep '\\[deploy pr\\]'")
-     def isDeployPR = result == 0 ? true : false
-     if (gitBranch == "master") {
-       env.DOCKER_TAG = "latest"
-     }
-     else if (isDeployPR) {
-        env.DOCKER_TAG = "${env.JOB_NAME?.split('/')[1]}-${env.BUILD_NUMBER}"
-        shortCommit = env.DOCKER_TAG+shortCommit
-     }
-     else {
-       env.DOCKER_TAG = env.TAG_NAME
-     }
+  node('buildSbtPod') {
+    sh "env | sort"
 
-     println "DOCKER_TAG: ${env.DOCKER_TAG}, gitDescribe: ${gitDescribe}, gitTag: ${gitTag}, commitId: ${commitId}, isDeployPR: ${isDeployPR}"
+    def repo = checkout scm
+    def gitBranch = repo.GIT_BRANCH
+    def gitCommit = repo.GIT_COMMIT
+    def shortCommit = repo.GIT_COMMIT[0..7]
+    def isDeployPR = sh(returnStdout: true, script: "git log -1").contains("[deploy pr]")
+    def taggedBranch = env.TAG_NAME != null
```

Comment:
Just to match other patterns (`GIT_` and `DOCKER_`), would it be better to use

`gitTag` vs `taggedBranch`

and 

`env.GIT_TAG` vs `env.TAG_NAME`?

Reply:
`env.TAG_NAME` is jenkins generated so we don't have much choice about that one and since `taggedBranch` is a bool instead of actually holding the tag I don't think it should be exactly like the others. How about `gitTagged`?

Reply:
👍